src/hooks/useContract.ts

```diff
@@ -92,6 +92,20 @@ export function useGovernanceContract(): Contract | null {
   )
 }
 
+export function useGovernanceContractBravo(): Contract | null {
```

Comment:
Why does this need its own hook? Can this logic be moved to the `useGovernanceContract` hook?